R/run_app.R

```diff
@@ -1,29 +1,41 @@
 #' Run the Shiny Application
 #'
-#' @param ... arguments to pass to golem_opts.
-#' See `?golem::get_golem_options` for more details.
+#' @param app_ver a "global" variable that is passed to several modules &
```

Comment:
I think maybe we should be using `{golem}` [configurations](https://cran.r-project.org/web/packages/golem/vignettes/e_config.html) to handle these "global" variables. Then we can be utilizing functions like `get_golem_config()`. This seems like a better approach than using `assign()`.

Reply:
Actually dug into this a little more, you can pass `app_version` as an option within `run_app()` for the argument `golem_opts` (i.e. `golem_opts = list(app_version = app_ver, ...)`). Then you can query this value in the code using `golem::get_golem_options('app_version')`.

A lot easier to implement than I thought. We should consider adding the database names and other values as `{golem}` options.